R/tm_file_viewer.R

```diff
@@ -90,6 +89,7 @@ tm_file_viewer <- function(label = "File Viewer Module",
   )
 }
 
+# UI function for the file viewer module
```

Comment:
@averissimo in other PRs @kartikeyakirar put `@noRd` and `@keywords internal` for those `ui_` and `server_` functions. Can we have them here as well, so it's unified... sorry?